app/mailers/year_to_date_report_mailer.rb

```diff
@@ -5,27 +5,28 @@ class YearToDateReportMailer < ApplicationMailer
 
   VA_STAKEHOLDERS = {
     to: %w[
-      Christopher.Marino2@va.gov
+      222A.VBAVACO@va.gov
       224B.VBAVACO@va.gov
+      224C.VBAVACO@va.gov
+      Anne.kainic@va.gov
+      Brandon.Scott2@va.gov
+      Brandye.Terrell@va.gov
       Carolyn.McCollam@va.gov
-      shay.norton@va.gov
       Christina.DiTucci@va.gov
-      Brandye.Terrell@va.gov
-      michele.mendola@va.gov
+      Christopher.Marino2@va.gov
       Christopher.Sutherland@va.gov
-      John.McNeal@va.gov
-      Anne.kainic@va.gov
       ian@adhocteam.us
-      Brandon.Scott2@va.gov
-      224C.VBAVACO@va.gov
-      peter.chou1@va.gov
+      John.McNeal@va.gov
       Joseph.Welton@va.gov
-      222A.VBAVACO@va.gov
-      Ricardo.DaSilva@va.gov
-      peter.nastasi@va.gov
-      Lucas.Tickner@va.gov
+      kathleen.dalfonso@va.gov
       kyle.pietrosanto@va.gov
+      Lucas.Tickner@va.gov
+      michele.mendola@va.gov
+      peter.chou1@va.gov
+      peter.nastasi@va.gov
+      Ricardo.DaSilva@va.gov
       robert.shinners@va.gov
+      shay.norton@va.gov
     ]
```

Comment:
Is this just alphabetized now?

Reply:
Yes, I figured that would make it easier to manage.